packages/tokens/src/color-primitives.yaml

```diff
@@ -189,3 +189,14 @@ color:
         value: hsl(45.1, 57.4%, 38.6%)
       900:
         value: hsl(45, 57.8%, 32.5%)
+  gradient:
+    yellowToCyan:
```

Comment:
We've been using kebab case for our names so perhaps `yellow-to-cyan` or `yellow-cyan` would work better?

Reply:
ah thanks for the catch, this was me just copy-pasta'ing 🙂 i'll update this